src/test/java/TestSenderBatteryData/TestSenderBatteryData.java

```diff
@@ -0,0 +1,34 @@
+package TestSenderBatteryData;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+import SenderBatteryData.CheckSenderData;
+import SenderBatteryData.SenderParam;
+import SenderBatteryData.SenderService;
+import SenderBatteryData.SenderServiceImpl;
+
+public class TestSenderBatteryData {
```

Comment:
Focus on testing your functionality - the most crucial is the output format. Did not see tests for that.
In the absence of a test for the output format, any future mistakes will be caught only while integrating with the receiver.